### DISCUSSION TITLE
Assignees based on packageRules

### DISCUSSION THREAD

**Discussion by firecow** [renovatebot/renovate#15417] [Request Help] [answered]
Hola, I'm trying to assign renovate MR's to specific personal depending on the matchManager package rules.

Project's renovate.json
```json
{
    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
    "assignees": ["jorgen"]
    "packageRules": [
        {
            "matchManagers": ["gitlabci-include"],
            "assignees": ["carsten"]
        }
    ]
}
```

Our global renovate config.js
```javascript
module.exports = {
  onboarding: false,
  dependencyDashboard: false,
  extends: [
    ":renovatePrefix",
    ":separateMultipleMajorReleases",    
    ":rebaseStalePrs",
    ":updateNotScheduled",
  ],
  packageRules: [
    {
      matchManagers: ["gomod"],
      groupName: "All gomod",
    }, 
    {
      matchManagers: ["docker-compose", "dockerfile"],
      groupName: "All docker",
    }, 
    {
      matchManagers: ["gitlabci-include"],
      groupName: "All gitlabci-include",
    }, 
    {
      matchManagers: ["npm"],
      groupName: "All npm",
    }, 
    {
      matchManagers: ["ansible-galaxy"],
      groupName: "All ansible-galaxy",
    }, 
    {
      matchManagers: ["composer"],
      groupName: "All php composer",
    }, 
    {
      matchManagers: ["maven"],
      groupName: "All maven",
    },
  ],
  regexManagers: [
    {
      fileMatch: ["^\.gitlab-ci\..*yml"],
      matchStrings: [".*?_IMAGE_URL: (?<depName>[\\w\\/.-]*?):(?<currentValue>[\\w.-]*)"],
      versioningTemplate: "regex:^(?<major>\\d+)(\\.(?<minor>\\d+))?(\\.(?<patch>\\d+))?(-(?<build>\\d*)(?<compatibility>.*))?$",
      datasourceTemplate: "docker",
    },
  ],
};

```

I've made sure that the project, has an outdated `gitlabci-include`.
Renovate doesn't make a MR at all about the outdated `gitlabci-include`.
If I remove the project specific packageRules, it makes a MR as it is supposed to. (THIS IS NOT TRUE)

How would you guys do this? 
The plan is to pass certain MR's to backenders, some to frontenders and others to pipeline responsible's.

**Comment by rarkins** [comment, marked as answer]
Assignees are added only at MR creation time, is that perhaps the problem? You can rename and close an existing MR and let Renovate create a replacement

**Reply by firecow** [reply]
I'm a giant idiot....

I purposely downgraded an include, that renovate had already upgraded, and that MR's obviously prevented a new MR from being created.

Sorry for wasting your time @rarkins 

The above configuration works just fine.

I've edited the abovte statement, it was not true, just me not understanding the workflow.
![image](https://user-images.githubusercontent.com/973602/166440174-2a59fee2-3b25-4b0b-b743-9faad8e895f4.png)
